test/README.md

```diff
@@ -0,0 +1,14 @@
+# RT-tests smoke tests 
+
+These are smoke tests that check if the main functionalities of each program is working.
+
+These tests are packaging agnostic, so doesn't matter how did you installed the `rt-tests` package.
```

Comment:
```suggestion
```
Actually they are packaging specific. For them to work, the snap needs aliases, and there may be other differences in other packaging forms which hasn't been tested.

Reply:
The requirement of having alias is snap specific, but not the tests itself. I can complement this talking that for the snap it is necessary to have alias.

Reply:
The test suite only works if rt-tests are installed as a deb. For the snap, aliases are needed. When built from source something else needs to be done (adding them to path). And possibly different for rpm, flatpack, etc that you haven't verified.

Reply:
When building from source and installed using the upstream `Makefile` it's installed on `/usr/local/bin` which already is in the PATH. But I understand what are you saying!